cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/parser/YamlParser.java

```diff
@@ -39,7 +41,15 @@ public ParsedSlang parse(SlangSource source) {
             parsedSlang.setName(source.getName());
             return parsedSlang;
         } catch (Throwable e) {
-            throw new RuntimeException("There was a problem parsing the YAML source: " + source.getName() + ".\n" + e.getMessage(), e);
+            String errorMessage = e.getMessage();
+            if (e instanceof ScannerException && errorMessage.startsWith("mapping values")){
+                errorMessage += "Probably did not provide (key: value) pair or missing space after colon(:)";
+            }
+            if (e instanceof ConstructorException && errorMessage.startsWith("Cannot create property")){
```

Comment:
I figured "e" cannot be two error types at once. But I can add "else if" just to be more clear